redis_elasticache/README.md

```diff
@@ -0,0 +1,54 @@
+# Elasticache Redis
+
+Module to deploy EC2 instance and required resources in a lazy fashion
+
+## Resources
+
+- Security Group
+- Tags
+- Subnets
+- Elasticache Cluster
+- Elasticache Parameter Group
+
+## Input Variables
+
+ | Name               | Type          | Default                                        | Example           |
+ | ------------------ | ------------- | ---------------------------------------------- | ----------------- |
+ | profile            | string        | default                                        |                   |
+ | domain_name        | string        |                                                | eng-cache         |
+ | family             | string        | redis6.x                                       |                   |
+ | node_type          | string        | cache.m6g.large                                | cache.m6g.xlarge  |
+ | engine_version     | string        | 6.x                                            |                   |
+ | maintenance_window | string        | sun:05:00-sun:09:00                            |                   |
+ | vpc_id             | string        |                                                | vpc-26r9f023fh2f3 |
+ | inbound_cidrs      | array(string) | ["0.0.0.0/0"]                                  |                   |
+ | user_tags          | object        |                                                | `see below`       |
+ | octopus_tags       | object        | (Required only if not running through octopus) |                   |
+
+## Example .tf file module reference
+
+```bash
+module "cache_cluster" {
```

Comment:
octopus_tags are missing in the reference module

Reply:
Yes I specifically didn't include them since they'll be added as environment variables.